packages/config/config/devices/0x013c/pat02-a.json

```diff
@@ -0,0 +1,338 @@
+// Philio Technology Corp PAT02-A
+// Flood Sensor
+{
+	"manufacturer": "Philio Technology Corp",
+	"manufacturerId": "0x013c",
+	"label": "PAT02-A",
+	"description": "Flood Sensor",
+	"devices": [
+		{
+			"productType": "0x0002",
+			"productId": "0x001f"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Dim Level (Off)",
+			"description": "When the flood trigger is off, send this dim level via the Basic CC to Group 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Basic Set Dim Level (On)",
+			"description": "When the flood trigger is on, send this dim level via the Basic CC to Group 2.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5[0x00]": {
+			"label": "Flood Sensor Function",
+			"description": "Enable or disable the flood function.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled",
+					"value": 0
+				}
+			]
+		},
+		"5[0x03]": {
+			"label": "Temperature Scale",
+			"description": "Configure the temperature scale reported by the device.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Fahrenheit",
+					"value": 0
+				},
+				{
+					"label": "Celsius",
+					"value": 1
+				}
+			]
+		},
+		"5[0x05]": {
```

Comment:
Bit 5? That needs to be 0x20 then

Reply:
Value 32 deaktivates temperaturreport

Reply:
Where are you getting 32? The manual says Bit 5?